tox.ini

```diff
@@ -82,5 +82,5 @@ commands = pytest -sv \
 
 [testenv:func]
 changedir = {toxinidir}/tests/functional
-commands = functest-run-suite {posargs:--keep-faulty-model}
```

Comment:
This is not useful if you want to do local deployment, because like this I could not set any parameters. 
Also, now the Makefile use FUNCARGS which are useless.

Reply:
@rgildein I believe this is temporary for debugging the CI failure in this PR.

Reply:
In that case, this should be draft PR.

Reply:
point taken, I should have converted to draft